nemo/collections/nlp/modules/common/sequence_classifier.py

```diff
@@ -36,15 +36,30 @@ def output_types(self) -> Optional[Dict[str, NeuralType]]:
 
     def __init__(
         self,
-        hidden_size: object,
-        num_classes: object,
-        activation: object = 'relu',
-        log_softmax: object = True,
-        dropout: object = 0.0,
-        num_layers: object = 1,
-        use_transformer_pretrained: object = True,
-    ) -> object:
+        hidden_size: int,
+        num_classes: int,
+        num_layers: int = 2,
+        activation: str = 'relu',
+        log_softmax: bool = True,
+        dropout: float = 0.0,
+        use_transformer_init: bool = True,
+        idx_conditioned_on: int = 0,
+    ):
+        """
+        Initializes the SequenceClassifier module.
+        Args:
+            hidden_size (int): the hidden size of the mlp head on the top of the encoder
+            num_classes (int): number of the classes to predict
+            num_layers (int)_layers (int): number of the linear layers of the mlp head on the top of the encoder
+            activation (str): type of activations between layers of the mlp head
+            log_softmax (bool): applies the log softmax on the output
+            dropout (float): the dropout used for the mlp head
+            use_transformer_init (bool): initializes the weights with the same approach used in Transformer
+            idx_conditioned_on (int): index of the token which its outputs are being used for classification, default is the first token
```

Comment:
Could you please rephrase the description, maybe smth like this: 'index of the token to use as the sequence representation for the classification task, default is the first token'

Reply:
Done